vortex-array/src/compute/search_sorted.rs

```diff
@@ -344,10 +345,8 @@ pub trait IndexOrd<V> {
     fn index_ge(&self, idx: usize, elem: &V) -> bool {
         matches!(self.index_cmp(idx, elem), Some(Greater | Equal))
     }
-}
 
-#[allow(clippy::len_without_is_empty)]
```

Comment:
this trait was a mistake, ide want to autosuggest it everywhere and it's not generally applicable